.github/actions/setup-deps/action.yaml

```diff
@@ -76,6 +76,8 @@ inputs:
     default: 'pytest-cov<=2.10.1'
   pytest-xdist:
     default: 'pytest-xdist'
+  pyedr:
```

Comment:
Are we making pyedr an optional dependency? I'm somewhat ok either way, if we could drop the 14MB test files in the next release it's small enough that it could be core if we wanted to. Thoughts @MDAnalysis/coredevs ?

Reply:
In any case, pyedr needs to go into the `setup.py` either as an extra_requires (if not a core dependency), or just as an install_requires otherwise

Reply:
If we can decrease the size to <1 MB then it's just more convenient to have it as a core dependency — it's light weight and we maintain it.